src/System.Diagnostics.FileVersionInfo/pkg/System.Diagnostics.FileVersionInfo.pkgproj

```diff
@@ -3,13 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Diagnostics.FileVersionInfo.csproj">
-      <SupportedFramework>net46;netstandardapp1.5</SupportedFramework>
+      <SupportedFramework>net46;netcore50;netstandardapp1.5</SupportedFramework>
```

Comment:
Good question, I don't think we have explicitly decided to support this on UWP but if metadata version works then great.